### PR TITLE
Separate function for Formplayer requests

### PR DESCRIPTION
**Pull request by wpride**
`post_data_helper` is doing some manipulation with the `urlparse()` object so add a new function modeled on the [validation endpoint](https://github.com/dimagi/commcare-hq/blob/3e88dbe4554316981e7f9a43fde34aa8870702c9/corehq/apps/nimbus_api/form_validation.py#L43-L43)

### CODE REVIEW

File: touchforms/formplayer/api.py [hunk 1]
```diff
@@ -279,24 +280,37 @@ def server_down(cls):
...
+
+
+def formplayer_post_data_helper(d, auth, content_type, url):
+    d['nav_mode'] = 'prompt'
+    data = json.dumps(d)
+    up = urlparse(url)
+    logging.info("Request to url: %s" % up.geturl())
+    headers = {}
+    headers["content-type"] = content_type
+    headers["Content-Type"] = content_type
```
Comment (gcapalbo) [inline touchforms/formplayer/api.py]: why is this here twice?

Comment (wpride) [reply touchforms/formplayer/api.py]: Copied over from the `validate_forms()` but I now read this is case-insensitive
